packages/scene-composer/src/components/toolbars/common/__snapshots__/ItemContainer.spec.tsx.snap

```diff
@@ -1,250 +1,40 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ItemContainer should render properly and trigger onClick when clicking on item 1`] = `
-.c0 {
```

Comment:
Randomly jumped to this file - looks like snapshots are not updating correctly. This content probably shouldn't be removed, and I see classnames are changed to something more random

Reply:
It's an intentional change from the jest version update to improve readability. We could re-enable it, however, since we use cloudscape I would suggest that we let them test their css. We're currently covered with e2e visual regression tests to ensure visual consistency.

Reply:
Validated locally that running the test for `SceneHierarchyTreeItem` six times did not create issues with mismatched snapshots.

Reply:
This snapshot is problematic because it's shouldn't be rendering the cloudscape component at all, the snapshot should look something like: "<div data-mocked="Container"><StuffWeCareAbout /></div>".

So somethings not mocking right, which means our tests can fail just because cloudscape updates their side.